src/structures/Presence.js

```diff
@@ -75,24 +75,24 @@ class Presence {
    * @readonly
    */
   get member() {
-    return this.guild.members.cache.get(this.userID) || null;
+    return this.guild.members.cache.get(this.userID) ?? null;
   }
 
   patch(data) {
     /**
      * The status of this presence
      * @type {PresenceStatus}
      */
-    this.status = data.status || this.status || 'offline';
+    this.status = data.status ?? this.status ?? 'offline';
 
     if (data.activities) {
       /**
        * The activities of this presence
        * @type {Activity[]}
        */
       this.activities = data.activities.map(activity => new Activity(this, activity));
-    } else if (data.activity || data.game) {
-      this.activities = [new Activity(this, data.game || data.activity)];
+    } else if (data.activity ?? data.game) {
+      this.activities = [new Activity(this, data.game ?? data.activity)];
```

Comment:
The access order in here isn't the same as of line above.

Reply:
tbh, wasn't `activity` and `game` removed on v8? can't this block just be removed?

Reply:
cc: @vladfrangu

Reply:
I don't seem to be receiving it anymore, it's also no longer in the [Discord Developer Documentation](https://discord.com/developers/docs/topics/gateway#presence-update).